example/django/bugsnag_demo/urls.py

```diff
@@ -1,13 +1,12 @@
 from django.conf.urls import url
-from demo.views import (index, crash, crash_with_callback, severity, notify,
+from demo.views import (index, crash, crash_with_callback, handled,
                         context, notify_meta)
 
 urlpatterns = [
     url(r'^$', index),
     url(r'^crash/$', crash),
-    url(r'^crash_with_callback/$', crash_with_callback),
-    url(r'^notify/$', notify),
-    url(r'^notify_meta/$', notify_meta),
-    url(r'^context/$', context),
-    url(r'^severity/$', severity),
+    url(r'^crashcallback/$', crash_with_callback),
+    url(r'^handled/$', handled),
+    url(r'^notifywithcontext/$', context),
+    url(r'^notifywithmetadata/$', notify_meta),
```

Comment:
The old example app seems to have a severity endpoint - is this concept demonstrated to users in one of the above URLs?

Reply:
Not sure about this, I'll query it.

Reply:
Yes,  the `context` endpoint now modifies the severity.